src/foam/comics/v2/DAOCreateView.js

```diff
@@ -68,13 +84,30 @@ foam.CLASS({
       }
     }
   ],
+  actions: [
+    {
+      name: 'save',
+      code: function() {
+        var self = this;
+        this.data = this.workingData;
+        this.data.createdBy = this.__subContext__.user.id;
```

Comment:
`createdBy` and `owner` shouldn't be filled in here.

Reply:
Removed createdBy,

But need to set owner otherwise I get the following error thrown upon creating an account:
"You do not have permission to create an account for another user."

I'm assuming the default/garbage value is 0 and so it doesn't get automatically set.

Reply:
You don't want that in this generic DAOCreateView though since it applies to any kind of object. We'll chat offline about this.

Reply:
Done. Removed it from the generic DAOCreateView and overrided the create function in the AccountDAOCreateView to add it in.